src/js/modules/StaticMap.js

```diff
@@ -0,0 +1,48 @@
+import React, {Component} from 'react';
+
+class StaticMap extends Component {
+
+  getLocation() {
+
+    const { street, city, state, zip, latitude, longitude } = this.props.location;
```

Comment:
Are we going to be getting lat and longs from Joplin?

Reply:
nvm, see you passing in some hardcoded lat/lons for other maps.

Reply:
yea, not entirely sure what we'll be getting so this is just flexible.